features/order/managing_orders/seeing_order_payment_state_as_completed_if_order_total_is_zero.feature

```diff
@@ -24,6 +24,6 @@ Feature: Seeing payment state as paid after checkout steps if order total is zer
     @ui
     Scenario: Seeing payment state as paid on order's summary
         When I view the summary of the order "#00000666"
-        Then it should have order's payment state "Paid"
-        And I should not see information about payments
+        And I should not see any payments
+        And I should see information about no payments
```

Comment:
`And I should be informed that there are no payments`?

Reply:
And remove previous step then